src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.scss

```diff
@@ -20,24 +24,28 @@
   font-weight: 300;
 }
 
+.previouslyViewedVersionedStableId {
+  margin-left: 8px;
```

Comment:
Notice the problem with the region: an empty span produces an extra space:

![image](https://user-images.githubusercontent.com/6834224/121611259-c74ef600-ca4f-11eb-861d-7915b9a64ef9.png)